### PR TITLE
UX: add color-scheme meta tag to _head

### PR DESCRIPTION
**Pull request by renato**
Adds the `color-scheme` meta tag to the `_head` partial and removes it from the finish installation template to prevent it from being added twice.

### CODE REVIEW

**Review by ZogStriP — APPROVED** [review]
👍

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -678,6 +678,13 @@ def discourse_theme_color_meta_tags
     result.html_safe
   end
 
+  def discourse_color_scheme_meta_tag
+    scheme = dark_scheme_id == -1 ? "light" : "light dark"
+    <<~HTML.html_safe
+        <meta name="color-scheme" content="#{scheme}">
+      HTML
```
Comment (davidtaylorhq) [inline app/helpers/application_helper.rb:685]: What if a site's default colour scheme is dark, and there is no alternate dark_scheme_id set?

(e.g. check the 'minima dark' theme on Meta)

Comment (renato) [reply app/helpers/application_helper.rb:685]: it should be covered, no?
https://github.com/discourse/discourse/blob/927d9696099f3482d9d31ce6ac66aaf60f7af08d/app/helpers/application_helper.rb#L557-L560

Comment (davidtaylorhq) [reply app/helpers/application_helper.rb:685]: If I visit meta as anon, and switch to the 'Minima Dark', then

- there is no dark_scheme_id cookie set
- There is no current_user
- There is no site setting configured

It's just that the 'default colour scheme' for the theme is dark.

So maybe something like:


```suggestion
    scheme = if dark_scheme_id == -1
      # no automatic client-side switching
      dark_color_scheme? ? "dark" : "light"
    else
      # auto-switched based on browser setting
      "light dark"
    end
    <<~HTML.html_safe
        <meta name="color-scheme" content="#{scheme}">
      HTML
```
(dark_color_scheme? is here: https://github.com/discourse/discourse/blob/ed13ae7787d5d915f1b22007a8b9ea9d71d4d257/app/helpers/application_helper.rb#L681C12-L684C6)

**Review by ZogStriP — APPROVED** [review]

**Pull request merged**